src/views/Analysis.vue

```diff
@@ -268,8 +267,7 @@ export default {
      */
     const chartType = useInitialOptions('chartType', { props, emit }, 'table')
 
-    /** @type {import('vue').Ref<HTMLElement>} template ref */
-    const toolbar = ref(null)
+    const toolbarRef = useTemplateRef('toolbar')
 
```

Comment:
This is somewhat tangential, but is a newer (clearer IMO) way of defining template refs in Vue 3.5